source/_components/binary_sensor.ferienapidotde.markdown

```diff
@@ -0,0 +1,45 @@
+---
+layout: page
+title: "ferien-api.de Binary Sensor"
+description: "Steps to configure the ferien-api.de sensor."
+date: 2019-03-24 12:30
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: home-assistant.png
```

Comment:
it is not intended that the logo of home assistant be used, when there is no logo for a component.